docs/src/atopics.md

```diff
@@ -1,5 +1,57 @@
 # Advanced Topics
 
+```@example 1
+import JSServe # hide
+JSServe.Page(exportable=true, offline=true) # hide
+```
+
+## Gradient field visualization
+
+FerriteViz also makes it easy to visualize gradient fields, like for example strain or stress fields.
+A common approach to visualize stresses and strains is to compute the L2 projection onto a H1 field and plot this.
+However, a big downside is that we loose the ability to investigate the jumps between elements, as they get smoothed out, hiding possible issues in the solution.
+Therefore, we provide the ability to interpolate the gradient into a piecewise discontinuous field via `FerriteViz.interpolate_gradient_field`.
+This function may be moved to Ferrite in the future.
+
+In this quick example we show how to visualize strains and stresses side-by-side
+```@example 1
+using Ferrite
+import FerriteViz
+using FerriteViz: ε
+import WGLMakie #activating the backend, switch to GLMakie or CairoMakie (for 2D) locally
+WGLMakie.set_theme!(resolution=(1500, 1600)) # hide
```

Comment:
```suggestion

```

I think this should be taken out. On my laptop this looks nice, but on my workstation with the large monitor it looks super odd, can you open the docs and make a screenshot for me in the best case on both - laptop and workstation monitor?